### PR TITLE
adding xvfb to support pyvista

### LINKED ISSUE #862: Add xvfb to default image so that pyvista can work
## Description

https://docs.pyvista.org/user-guide/jupyter/panel.html#configuration-considerations

### PR DESCRIPTION
**Pull request by rsignell-usgs**
Fixes #862

## Changes introduced in this PR:

adding xvfb as a system library to support pyvista

## Types of changes

What types of changes does your PR introduce?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

### Requires testing

- [ ] Yes
- [x] No

## Documentation

No doc changes necessary.

### CODE REVIEW

**Review by iameskild — APPROVED** [review]
Hey @rsignell-usgs, thanks for the contribution! Given how small these packages are (adding up to less than 2.5mb, [`xvfb`](https://packages.ubuntu.com/focal/xvfb), [`lib..`](https://packages.ubuntu.com/focal/libgl1-mesa-dev)), I'm with adding them to the general image for now. We'll probably be cleaning these images up in the future so a larger discussion around what to include will surely follow :)

Thanks again!

cc @costrouc @viniciusdc 

PS - the failed CI is because this is coming from a branch on a fork. The kubernetes test passed and that's a good sign.

File: qhub/template/image/jupyterlab/apt.txt [hunk 1]
```diff
@@ -27,3 +27,6 @@ libxcomposite1
 libasound2
 libxi6
 libxtst6
+
+# pyvista
+xvfb
```
[the remark this answers is not in the document]
Comment (viniciusdc) [reply qhub/template/image/jupyterlab/apt.txt:32]: I think this should be added after `v0.4.3`, this way we can check if there is anything else that `xvfb` might require. @iameskild

**Comment by viniciusdc** [comment]
> @iameskild, Ready to merge now that 4.3 is out?

Hi @rsignell-usgs, apologies for the late reply. As we are in the migration process from Qhub to Nebari, our docker images were moved to a separate repository under the Nebari org. I have already created a PR to add this, which will be part of the 0.4.4 release.

**Comment by iameskild** [comment]
Closing in favor of: https://github.com/nebari-dev/nebari-docker-images/pull/20

**Pull request closed without merging**